ci/linux-debian.Dockerfile

```
@@ -10,4 +10,8 @@ RUN apt-get install --no-install-recommends --no-upgrade -y \
         make automake libtool pkg-config dpkg-dev valgrind qemu-user \
         gcc clang libc6-dbg \
         gcc-i686-linux-gnu libc6-dev-i386-cross libc6-dbg:i386 \
-        gcc-s390x-linux-gnu libc6-dev-s390x-cross libc6-dbg:s390x
+        gcc-s390x-linux-gnu libc6-dev-s390x-cross libc6-dbg:s390x \
+        wine gcc-mingw-w64-x86-64
```

(not strictly necessary but probably cleaner:)
```suggestion
        wine64 gcc-mingw-w64-x86-64
```